pkg/util/http.go

```diff
@@ -19,17 +19,34 @@ package util
 import (
 	"crypto/tls"
 	"net/http"
+	"os"
 	"time"
 )
 
 var disableKeepAlives bool
+var minTLSVersion uint16
 
 func init() {
 	var err error
 	disableKeepAlives, err = ResolveOsEnvBool("KEDA_HTTP_DISABLE_KEEP_ALIVE", false)
 	if err != nil {
 		disableKeepAlives = false
 	}
+
+	minTLSVersion = tls.VersionTLS12
+	version, found := os.LookupEnv("KEDA_HTTP_MIN_TLS_VERSION")
+	if found {
+		switch version {
+		case "TLS13":
+			minTLSVersion = tls.VersionTLS13
+		case "TLS12":
+			minTLSVersion = tls.VersionTLS12
+		case "TLS11":
+			minTLSVersion = tls.VersionTLS11
+		default:
+			minTLSVersion = tls.VersionTLS10
```

Comment:
Shouldn't this be `minTLSVersion = tls.VersionTLS12`?

Reply:
I added it to not fail with wrong versions. I mean, if the env isn't set, we will use `VersionTLS12`. Maybe I can add the case TLS10 and set default to VersionTLS12 with a message, let me chagne it

Reply:
I added it to not fail with wrong versions. I mean, if the env isn't set, we will use `VersionTLS12`. Maybe I can add the case TLS10 and set default to VersionTLS12 with a message, let me change it

Reply:
Done, PTAL